examples/demo/src/lib.rs

```diff
@@ -31,7 +31,7 @@ pub fn demo_element<'a>(cx: Scope<'a>) -> Element<'a> {
                                 padding_bottom: 30,
                                 bar_width: "10%",
                                 horizontal_bars: true,
-                                label_interpolation: |v| format!("{v}%"),
+                                label_interpolation: (|v| format!("{v}%")) as fn(f32) -> String,
```

Comment:
Can't you just do?:
```suggestion
                                label_interpolation: |v: f32| format!("{v}%"),
```

Reply:
Same for the test

Reply:
Unfortunately not. According to JK, it's a Rust issue. See [here](https://discord.com/channels/899851952891002890/943190605067079712/1219870965190758482)